backend/api/Areas/Admin/Controllers/AccessRequestController.cs

```diff
@@ -33,7 +33,7 @@ public class AccessRequestController : Controller
         /// </summary>
         /// <param name="pimsService"></param>
         /// <param name="mapper"></param>
-        public AccessRequestController(IPimsService pimsService, IMapper mapper)
+        public AccessRequestController(IPimsRepository pimsService, IMapper mapper)
```

Comment:
nit: I know this work is still in flux but would prefer if naming was consistent; i.e. `IPimsRepository pimsRepository`

Reply:
Yes, this was due to my decision to try and fit this into the sprint without it being planned. Ibudgeted some time for these changes and used up all of that budget without 100% completing everything I wanted, like renaming all of the existing services.

I made a call that the partial change was still worth keeping as it enables future services/repositories to be developed in the manner that we want. The downside is leftover naming like this.

Reply:
Also, that change has already been merged, I'm in the process of updating a PR to only reflect the changes from psp-2020. Should be done shortly.